lsp/Main.hs

```diff
@@ -248,6 +250,24 @@ instance HasLoc t => HasLoc (SomeAstNode t) where
   getLoc (SClassDecl et) = getLoc et
   getLoc (SGlobalVarDecl et) = getLoc et
   getLoc (SRule et) = getLoc et
+  getLoc (SAssertion et) = getLoc et
```

Comment:
This could be implemented on top of `getAnnot` now (and simplified to a single line)